chrome/content/zotero/xpcom/translation/translate.js

```diff
@@ -2182,15 +2188,22 @@ Zotero.Translate.Export.prototype._prepareTranslation = function() {
 	this._itemGetter = new Zotero.Translate.ItemGetter();
 	var configOptions = this._translatorInfo.configOptions || {},
 		getCollections = configOptions.getCollections || false;
-	if(this._collection) {
-		this._itemGetter.setCollection(this._collection, getCollections);
-		delete this._collection;
-	} else if(this._items) {
-		this._itemGetter.setItems(this._items);
-		delete this._items;
-	} else {
-		this._itemGetter.setAll(getCollections);
+	if (!this._export) { throw new Error('No export set up'); }
```

Comment:
JavaScript engine throwing "cannot read property '_export' of null" is good enough here, since we get a stack anyway.